### PR TITLE
Update dependencies location ++ Specify the values location ++ Not use `helm create`

### LINKED ISSUE #2: Review where the dependencies should be declared within the usage section
## To do

Can you please review the `README usage` as the dependencies should be now declared part of the Chart.yaml file apparently

```
helm dependency update
load.go:120: Warning: Dependencies are handled in Chart.yaml since apiVersion "v2". We recommend migrating dependencies to Chart.yaml.
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "http://snowdrop.github.io/helm" chart repository
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "snowdrop" chart repository
...Successfully got an update from the "k8s-dashboard" chart repository
...Successfully got an update from the "harbor" chart repository
...Successfully got an update from the "traefik" chart repository
...Successfully got an update from the "stratos" chart repository
...Successfully got an update from the "grafana" chart repository
...Successfully got an update from the "kiwigrid" chart repository
...Successfully got an update from the "jenkins" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading spring-boot-example-app from repo http://snowdrop.github.io/helm
Deleting outdated charts
```

See helm v3 doc - https://helm.sh/docs/helm/helm_dependency/#helm

### LINKED ISSUE #3: To be improved: Configure the template accordingly using the chart name as root element
## To be improved

This section of the README is not clear as you dont explain in which file the information should be added

```
Configure the template accordingly using the chart name as root element:

spring-boot-example-app: # This is the chart name from the dependencies!
  name: rest-http
  version: 2.5.0-0-SNAPSHOT
  s2i:
    source:
      repo: https://github.com/snowdrop/rest-http-example
      ref: sb-2.5.x
```

### LINKED ISSUE #4: Wrong image repository declared - nginx
## Issue

I followed the instructions (created a helm project, added as dependency our snowdrop example and updated the dependency)
```
helm dependency update
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "http://snowdrop.github.io/helm" chart repository
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "snowdrop" chart repository
...Successfully got an update from the "k8s-dashboard" chart repository
...Successfully got an update from the "traefik" chart repository
...Successfully got an update from the "stratos" chart repository
...Successfully got an update from the "kiwigrid" chart repository
...Successfully got an update from the "harbor" chart repository
...Successfully got an update from the "jenkins" chart repository
...Successfully got an update from the "grafana" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading spring-boot-example-app from repo http://snowdrop.github.io/helm
Deleting outdated charts
```

Surprisingly if I open the `values.yaml` file the image repository declared is for a nginx app
```
image:
  repository: nginx
...
```

### PR DESCRIPTION
**Pull request by Sgitario**
Fix https://github.com/snowdrop/helm/issues/2
Fix https://github.com/snowdrop/helm/issues/3
Fix https://github.com/snowdrop/helm/issues/4

### CODE REVIEW

**Comment by Sgitario** [comment]
@cmoulliard about https://github.com/snowdrop/helm/issues/4, this is a consequence of using `helm create` which does not allow to create an empty/blank helm structure. I changed the instructions to not use it. Let me know what you are ok with.

File: README.md [hunk 1]
```diff
@@ -43,24 +43,35 @@ snowdrop	    https://snowdrop.github.io/helm
 To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
 
 ```console
-$ helm create rest-http
+$ mkdir rest-http
 ```
-This command will generate the following directory structure using th Helm starter template:
+
+Now, let's create the `Chart.yaml` file at `rest-http/` with your Chart information:
```
Comment (cmoulliard) [inline README.md]: ... `file at `rest-http/` ... ==> ... under the `rest-http/` folder

File: README.md [hunk 2]
```diff
@@ -43,24 +43,35 @@ snowdrop	    https://snowdrop.github.io/helm
...
+
+```yaml
+apiVersion: v2
+name: rest-http
+description: A Helm chart for Kubernetes
+```
+
+And let's also create the `values.yaml` file at `rest-http/` and keep it empty for now.
```
Comment (cmoulliard) [inline README.md]: Same remark as before

File: README.md [hunk 3]
```diff
@@ -43,24 +43,35 @@ snowdrop	    https://snowdrop.github.io/helm
...
 │   Chart.yaml
 │   values.yaml
-|
-└───charts
-└───templates
-│   │   ...
 ```
 
-To update the created project using the Helm resource defined the example, we will then create the [dependencies file](https://v2.helm.sh/docs/charts/#chart-dependencies) under the root path of the project `rest-http/requirements.yaml`.
+To update the created project using the Helm resource defined the example, we will then add the [chart dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies) in `rest-http/Chart.yaml`.
```
Comment (cmoulliard) [inline README.md]: I dont understand this text `To update the created project using the Helm resource defined the example`. To be rephrase

**Review by cmoulliard — COMMENTED** [review]
This text `And, install your helm:` should be changed to `Next or Finally, install your helm chart:`

File: README.md (hunk 1 above)
Comment (Sgitario) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (Sgitario) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (Sgitario) [reply README.md]: Rephrased

**Comment by Sgitario** [comment]
@cmoulliard PR updated with your comments.

File: README.md [hunk 4]
```diff
@@ -43,24 +43,34 @@ snowdrop	    https://snowdrop.github.io/helm
...
 │   values.yaml
-|
-└───charts
-└───templates
-│   │   ...
 ```
 
-To update the created project using the Helm resource defined the example, we will then create the [dependencies file](https://v2.helm.sh/docs/charts/#chart-dependencies) under the root path of the project `rest-http/requirements.yaml`.
-Next, edit the file to specify the dependency to be used
+Next, to use the `spring-boot-example-app` dependency, we need to add the [dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies) section in `rest-http/Chart.yaml`:
```
Comment (cmoulliard) [inline README.md]: `to add ... section in rest-http/Chart.yaml:` --> `... append ... section within the rest-http/Chart.yaml file:`

File: README.md [hunk 5]
```diff
@@ -72,10 +82,10 @@ When done, execute this command to download the dependency:
 $ helm dependency update
 ```
 
-Configure the template accordingly using the chart name as root element:
+Edit the `rest-http/values.yaml` file to configure the dependency `spring-boot-example-app`:
```
Comment (cmoulliard) [inline README.md]: Next, edit the `rest-http/values.yaml` file to configure the dependency to be used such as `spring-boot-example-app`:

File: README.md (hunk 4 above)
Comment (Sgitario) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (Sgitario) [reply README.md]: done

**Pull request merged**